pallets/staking/src/lib.rs

```diff
@@ -2644,6 +2646,32 @@ decl_module! {
                     .map(|p| p.intended_count = new_intended_count)
             })
         }
+
+
+        /// Declare no desire to either validate or nominate.
+        ///
+        /// Effects will be felt at the beginning of the next era.
+        ///
+        /// The dispatch origin for this call must be a GC.
+        /// And, it can be only called when [`EraElectionStatus`] is `Closed`.
+        #[weight = <T as Config>::WeightInfo::chill_from_governance()]
+        pub fn chill_from_governance(origin, identity: IdentityId, stash_keys: Vec<T::AccountId>) {
+            ensure!(Self::era_election_status().is_closed(), Error::<T>::CallNotAllowed);
+            let _controller = ensure_signed(origin)?;
+            PermissionedIdentity::remove(&identity);
```

Comment:
Need to also check that the identity is a permissioned validator.
```suggestion
            ensure!(Self::permissioned_identity(&identity).is_some(), Error::<T>::NotExists);
            // Change identity status to be Non-Permissioned
            PermissionedIdentity::remove(&identity);
```